.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This is simply to avoid a conflict issue with `babel-jest`.  Will be able to remove this when the new version of that dependency gets released.